packages/react-ui-map/src/Mapbox/Mapbox.js

```diff
@@ -83,17 +89,19 @@ export default class Mapbox extends PureComponent {
     const { map } = this.state
 
     return (
-      <div
-        style={{}}
-        id="map"
-        className={classNames(
-          className,
-          theme[`Map-${color}`],
-          theme[`Map-${size}`],
-          theme.Map
-        )}
-      >
-        {map && children}
+      <div>
+        <div
+          ref="map"
+          className={classNames(
+            className,
+            theme[`Map-${color}`],
+            theme[`Map-${size}`],
+            theme.Map,
+          )}
+        >
+          {map && children}
+        </div>
+        <button className={theme.Button} onClick={this.handleClick}>Fit</button>
```

Comment:
Please put these props on their own new lines. Also, why is this button `Fit` in this map file in the first place?  Is it supposed to be here???